CHANGELOG.md

```diff
@@ -56,6 +56,30 @@ Read our [guidelines for writing a good changelog entry](https://github.com/biom
 
 ### Configuration
 
+#### New features
+
+- Add the ability to resolve the configuration files defined inside `extends` from the `node_modules/` directory.
+
+  If you want to resolve a configuration file that matches the specifier `@org/configs/biome`, then your `package.json` file must look this:
```

Comment:
Can we add a link to Node.js resolver algorithm?

Reply:
Where and how? Feel free to add a suggestion. I don't know where that page is

Reply:
I added it in the documentation page